src/app/utils/numberFormatter.ts

```diff
@@ -0,0 +1,42 @@
+import { BigNumber } from 'bignumber.js'
+
+export type NumberFormattingParameters = Partial<BigNumber.Format> & {
+  // Additional features which are not natively supported by BigNumber
+
+  decimalPlaces?: number
+  maximumFractionDigits?: number
+  roundingMode?: BigNumber.RoundingMode
+  unit?: string
+}
+
+export const formatNumber = (
+  inputNumber: number | string | BigNumber.Instance | undefined,
+  format: NumberFormattingParameters = {},
```

Comment:
If the future we'll probably need to add formatting based on locale.

Some approaches for that:
- Param: `formatNumber(111, { useTranslation() })`
- Hook: `const formatNumber = useFormatNumber()` and internally use useTranslation
- i18n: Does i18n have a way to expand `{{value, number}}` with custom `{{value, bignumber}}`

Reply:
OK, using a hook, for now.